backend/src/main/java/gov/cdc/usds/simplereport/service/errors/InvalidAddressException.java

```diff
@@ -0,0 +1,10 @@
+package gov.cdc.usds.simplereport.service.errors;
+
+public class InvalidAddressException extends RuntimeException {
+
+  private static final long serialVersionUID = 1L;
+
+  public InvalidAddressException(String message) {
```

Comment:
Does this really need to take a message as a constructor parameter? Isn't the message always going to be the same, because the use case for this exception is always the same?